scripts/Interface/reduction_gui/instruments/instrument_factory.py

```diff
@@ -1,3 +1,4 @@
+#pylint: disable = W0403
```

Comment:
Mantid might be moving to Python 3 at some point, so instead of disabling these warning, the relative imports should be fixed (changed to absolute or declared explicitly as relative), as Python 3 does not handle implicit relative imports.